workflow/artifacts/git/git_test.go

```diff
@@ -13,24 +13,29 @@ import (
 var d = uint64(1)
 
 func TestGitArtifactDriver_Load(t *testing.T) {
-	_ = os.Remove("git-ask-pass.sh")
-	driver := &ArtifactDriver{}
-	path := "/tmp/git-found"
-	assert.NoError(t, os.RemoveAll(path))
-	assert.NoError(t, os.MkdirAll(path, 0o777))
-	err := driver.Load(&wfv1.Artifact{
-		ArtifactLocation: wfv1.ArtifactLocation{
-			Git: &wfv1.GitArtifact{
-				Repo:     "https://github.com/argoproj/argoproj.git",
-				Fetch:    []string{"+refs/heads/*:refs/remotes/origin/*"},
-				Revision: "HEAD",
-				Depth:    &d,
+	for _, url := range []string{
+		"https://github.com/argoproj/argo-workflows.git",
+		"https://github.com/tczhao/empty-repo.git",
```

Comment:
maybe we need a argoproj empty repo?

Reply:
https://github.com/argoproj/empty

Reply:
since its a private repo

added to
- TestGitArtifactDriverLoad_SSL
- TestGitArtifactDriverLoad_HTTPS

skip
- TestGitArtifactDriver_Load